strongbox-storage/strongbox-storage-layout-providers/strongbox-storage-maven-layout/strongbox-storage-maven-layout-provider/src/main/java/org/carlspring/strongbox/storage/indexing/RepositoryIndexFounder.java

```diff
@@ -0,0 +1,28 @@
+package org.carlspring.strongbox.storage.indexing;
+
+import org.carlspring.strongbox.providers.io.RepositoryPath;
+import org.carlspring.strongbox.storage.repository.Repository;
+import org.carlspring.strongbox.storage.repository.RepositoryTypeEnum;
+import org.carlspring.strongbox.util.ThrowingFunction;
+
+import javax.inject.Qualifier;
+import java.io.IOException;
+import java.lang.annotation.Retention;
+
+import static java.lang.annotation.RetentionPolicy.RUNTIME;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+public interface RepositoryIndexFounder
```

Comment:
Cosmetic: This class' name is incorrect. In this case,`found` is used in the sense of `locate`.

You can have a look at [this](https://www.merriam-webster.com/dictionary/founder) for clarifications.

I suggest we call these locators. Could you please appluy this change to the implementing class(es)? Thanks! :)

Reply:
> Cosmetic: This class' name is incorrect. In this case,found is used in the sense of locate.

@carlspring as I understand this seems not `locate` as well, this assumed to create index files, but not just locate them. Anyway this probably better would be the `RepositoryIndexCreator`.

Reply:
Changed to `*Creator`